tox.ini

```diff
@@ -3,6 +3,7 @@ envlist =
     lint,
     check-migrations,
     test,
+    test-docker,
```

Comment:
@AlmightyYakob is there a reason to not just do this and get rid of the `test-docker` workflow?

Reply:
No I think that's fine. I was originally thinking I'd run these separately (like running the regular tests always, but only running the docker tests once in a while). However, they're not that slow to run, so I think this is fine.